Automattic/wp-calypso
# Collect data from site goals field

A continuation of https://github.com/Automattic/wp-calypso/pull/20078 and https://github.com/Automattic/wp-calypso/pull/19693. This PR adds some infrastructure to track and record values checked off under the goal question for our new user segmenting step in signup (more details here: p5XAZ9-1Cn-p). I will create yet another PR to process the results in a bid to wrap up work for this field. 

![image](https://user-images.githubusercontent.com/6981253/33145198-7b1bd6d8-cf8d-11e7-8344-8b30112c2437.png)

## Testing
* Visit `/start/segment`
* Select some checkbox options
* Click `Continue`
* Click `Back` at the bottom of the next screen to see if your options remained in tact. 
* Rinse and repeat with other steps further in the funnel.

@markryall and @taggon can you take a look?
